go/ct/rlz/conditions_test.go

```diff
@@ -288,3 +288,55 @@ func TestHasSelfDestructedCondition_HasSelfDestructRestrictsGeneratedStateToBeSe
 		t.Error("generated state does not satisfy condition")
 	}
 }
+
+func TestValidRange256_Check(t *testing.T) {
```

Comment:
This is a very high-level end-to-end test for the new conditions. It would be good to test Check, Restrict, and GetTestValues independently as well.